datadog_checks_base/datadog_checks/base/utils/tagging.py

```diff
@@ -0,0 +1,20 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+try:
+    import tagger
+except ImportError:
+    from ..stubs import tagger
+
+try:
+    # Try to access the 6.11+ API
+    tagger.tag("", tagger.ORCHESTRATOR)
+except AttributeError:
+    # 6.10 or lower, add a translation layer
```

Comment:
Do we need to add this ? base check is supposed to work with a fixed version of the agent. 
Unless it's for agent 5 ?

Reply:
indeed, that would be only for backwards compat. Shall we delete it then?